Documentation/Storage-Configuration/NFS/nfs-security.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Security
+---
+
+## User ID management
+
+!!! attention
```

Comment:
We only have attention notes in this section. Is there some summary of ID management to include as well?

Reply:
Good point. I'll add a simple overview.